jplag/src/main/java/de/jplag/options/LanguageOption.java

```diff
@@ -34,7 +35,7 @@ public String getDisplayName() {
     }
 
     public static LanguageOption fromDisplayName(String displayName) {
-        return Arrays.stream(LanguageOption.values()).filter(languageOption -> languageOption.displayName.equals(displayName)).findFirst()
+        return Arrays.stream(LanguageOption.values()).filter(languageOption -> languageOption.displayName.equalsIgnoreCase(displayName)).findFirst()
```

Comment:
Why this change ?

Reply:
This came up in a discussion above. While this could be a separate PR. I think we can include this here. Maybe we should mention it in the PR description though.

Reply:
I updated the description.